llvm/lib/Transforms/IPO/DynUntrustedAllocPost.cpp

```diff
@@ -380,7 +568,10 @@ class DynUntrustedAllocPost : public ModulePass {
     llvm::raw_fd_ostream OS(PreStats->FD, /* shouldClose */ false);
     OS << "Number of alloc instructions modified to unsafe: "
        << modified_inst_count << "\n"
-       << "Total number hooks given a UniqueID: " << total_hooks << "\n";
+       << "Total number hooks given a UniqueID: " << total_hooks << "\n"
```

Comment:
UniqueID doesn't seem to make sense anymore ...

Reply:
Switched to localID naming.